### PR TITLE
[wip] Hugo/teams

### PR DESCRIPTION
**Pull request by dkhgh**

### CODE REVIEW

File: spec/models/team_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (fatboypunk) [inline spec/models/team_spec.rb]: only require spechelper

File: spec/features/user_visits_teams_overview_spec.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+feature "user visits teams overview" do
+  before :each do
+    sign_in_user
+  end
+
+  scenario "user visits teams overview" do
+    click_link "Teams"
+    expect(page).to have_content("All Teams")
+  end
+end
```
Comment (fatboypunk) [inline spec/features/user_visits_teams_overview_spec.rb]: @tarzan can you share your end of line sublime config with @dkhgh ?

**Comment by fatboypunk** [comment]
Rebase on development :)

File: app/models/user.rb [hunk 3]
```diff
@@ -5,13 +5,38 @@ class User < ActiveRecord::Base
...
   def self.create_with_omniauth(auth)
     create! do |user|
       user.provider = auth["provider"]
       user.uid = auth["uid"]
       user.name = auth["info"]["name"]
       user.email = auth["info"]["email"]
+
+      domain = user.email.split("@").last
+
+      if Team.where(domain: domain).exists?
```
Comment (dkhgh) [inline app/models/user.rb:32]: should we put this in an operation?

**Comment by fatboypunk** [comment]
How far are we with this?

**Pull request merged**